### ISSUE TITLE
[BUDI-7579] Title setting within settings -> branding doesn't seem to work

### ISSUE THREAD

**Issue by ConorWebb96** [Budibase/budibase#11940]
**Checklist**
- [x] I have searched budibase discussions and github issues to check if my issue already exists

**Hosting**
- Self
  - Method: docker compose
  - Budibase Version: 2.10.10
  - App Version: 2.10.10

**Describe the bug**
The title input field within the settings -> branding page doesn't seem to update the title of the browser's tab after changing the text and clicking the save button. This seems to be specific to self-host as it works on cloud.

**To Reproduce**
1. Go to the settings -> branding page
2. Change the Budibase text to your own custom text and click the save button
4. See error (Nothing changes)

**Expected behavior**
Whenever the text within the title input on this page is change and then saved. The title of the browser's tab should update to whatever the text appearing inside this input shows.

**Screenshots**
![Screenshot_2023-10-02_at_09 00 26](https://github.com/Budibase/budibase/assets/126772285/45459b15-5162-4832-baba-9ee46d02410d)

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Brave
 - Version v1.58.135

<sub>[BUDI-7579](https://linear.app/budibase/issue/BUDI-7579/title-setting-within-settings-branding-doesnt-seem-to-work)</sub>

**Comment by deanhannigan** [comment]
This appears to work as expected in my local environment. I didn't run into any errors when saving the title. I changed the title to `Dean2` and it appeared as expected in the title for the auth screen.

![Screenshot 2023-11-06 at 17.52.49.png](https://uploads.linear.app/7ecfa297-cc77-4fdf-9117-fa5afdb8f908/e1d89f8b-8882-4a8b-b4cc-f86788d84414/0564d216-b9b4-437b-9676-c54385853c34)

I added a similar change in prod and the title appeared in the tab too.

![Screenshot 2023-11-06 at 17.53.14.png](https://uploads.linear.app/7ecfa297-cc77-4fdf-9117-fa5afdb8f908/4bc63cad-77c2-4d02-8195-cfbd87d3ceb0/43df5094-f0af-46f9-a0a3-4fb208988159)
